Docs/source/usage/parameters.rst

```diff
@@ -797,6 +797,10 @@ Particle initialization
       If set to 4, symmetrization is in the x and y direction, (x,y) (-x,y) (x,-y) (-x,-y).
       If set to 8, symmetrization is also done with x and y exchanged, (y,x), (-y,x), (y,-x), (-y,-x)).
 
+      * ``<species_name>.focal_distance`` (optional, distance between the beam centroid and the position of the focal plane of the beam, along the direction of the beam mean velocity; space charge is ignored in the initialization of the particles)
+
+      If ``<species_name>.focal_distance`` is specified, ``x_rms``, ``y_rms`` and ``z_rms`` are the size of the beam in the focal plane. Since the beam is not necessarily initialized close to its focal plane, the initial size of the beam will differ from ``x_rms``, ``y_rms``, ``z_rms``.
```

Comment:
This is awesome.

In accelerator physics, people often think in terms of twiss parameters.

Can you potentially document here in the docs, for axis of prop is z, the relation between these inputs and alpha + beta + emittance?
@aeriforme